fenix/app/src/main/java/org/mozilla/fenix/compose/BetaLabel.kt

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import mozilla.components.ui.colors.PhotonColors
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Beta label.
+ */
+@Composable
+fun BetaLabel() {
```

Comment:
> Do you see it being reused very soon?

We're aiming to ship Translations in 122 (think of this as before or early Dec). 

1. I totally get this point since I have also read the article before. Just not sure if the `Modifier` should be applied to the `Card` or `Text`. One of the tricky thing is that the Beta label will look exactly the same for both use cases.

2. I would propose this as a follow up since this sounds like something we need to define and refactor from many other components. However, I am not aware of any standardized shapes that are defined by the design system. So, I am wondering if there is something out there that defines this for us or how do we intend to approach this? From my point of view, it would be easier if the design system said this "small card" shape has `RoundedCornerShape(8.dp)` and if they ever change the definition of "small card", then everything changes in a unified way.

3. I think this is one of the tricky bits. It's not exactly a design component, but it is a component that looks the same for both shopping and translations and can probably be reused by both. We could definitely ask @cwzilla to turn this into a design component, but that will require a followup.

Reply:
1. Gotcha, I would say Card since it's the top level composable here.
2. Follow up sounds good, I mentioned it more for us to think about it going forward. Material design has shapes - small, medium, large, if those 3 work for us, we can use those semantics and update the corder radii. 
3. Yeah I think the intention here is good, I think you already know but it's just that we should make sure design system has this component so it changes in figma in a uniform way that's all. Saving us from re-implementing in case one feature diverges since it wasn't in the design system.